modules/planning/planner/open_space/open_space_planner.cc

```diff
@@ -64,40 +66,32 @@ apollo::common::Status OpenSpacePlanner::Plan(
   init_v_ = init_state_.linear_velocity();
   Eigen::MatrixXd x0(4, 1);
   x0 << init_x_, init_y_, init_phi_, init_v_;
-
   // std::vector<double> x0({-12, 11, 0, 0});
 
   // final state
-
   // TODO(QiL): Step 2 ： Take final state from decision / or decision level
   // when enabled.
-
   Eigen::MatrixXd xF(4, 1);
   xF << 0, 1.2, M_PI / 2, 0;
-
   // std::vector<double> xf({0, 1.2, M_PI / 2, 0});
 
-  // TODO(QiL): Step 3 : Get obstacles from map/perception convex sets from
-  // vetices using H-represetntation
-  std::size_t nOb = 3;  // number of obstacles
-  // vetices of each obstacles
-  Eigen::MatrixXd vOb(3, 1);
-  vOb << 4, 4, 4;
-
-  // TODO(QiL) : Clean up, represent lOb with better format
-  // vetices cw presentation
-
-  // std::vector<std::vector<Eigen::MatrixXd>> lOb = {{[-20; 5]}};
-  std::vector<std::vector<std::vector<double>>> lOb = {
-      {{-20, 5}, {-1.3, 5}, {-1.3, -5}, {-20, -5}, {-20, 5}},
-      {{1.3, 5}, {20, 5}, {20, -5}, {1.3, -5}, {1.3, 5}},
-      {{-20, 15}, {20, 15}, {20, 11}, {-20, 11}, {-20, 15}}};
-  // TODO(QiL) : more efficient way of initialize lOb in open space planner
-  //  std::vector<std::vector<Eigen::MatrixXd>> lOb;
-  Eigen::MatrixXd ob1(4, 1), ob2(4, 1), ob3(4, 1);
-  ob1 << -1.3, 5, 20, 5;
-  ob2 << 20, 5, -1.3, 5;
-  ob3 << 20, 15, 20, -11;
+  // vertices using V-represetntation (clock wise)
+  std::size_t obstacles_num = frame->obstacles_num();
+  Eigen::MatrixXd obstacles_vertices_num = frame->obstacles_vertices_num();
+  std::vector<std::vector<Vec2d>> obstacles_vertices_vec =
+      frame->obstacles_vertices_vec();
+  Eigen::MatrixXd obstacles_A = frame->obstacles_A();
+  Eigen::MatrixXd obstacles_b = frame->obstacles_b();
+  // std::vector<std::vector<std::vector<double>>> obstacles_vertices_vec = {
```

Comment:
please remove comments

Reply:
still need that. will delete in the future